.htmlhintrc

```diff
@@ -0,0 +1,61 @@
+{
+  /*
+   * STANDARD
+   * =================
+   */
+  
+  // All tagnames must be in lowercase only
+  "tagname-lowercase": true,
+  
+  // All attribute names must be in lowercase
+  "attr-lowercase": true,
+  
+  // Attribute values must be enclosed by double quotemarks
+  "attr-value-double-quotes": true,
+  
+  // Attributes cannot be duplicated on a single tag
+  "attr-no-duplication": true,
+  
+  // Don't look for doctypes in the html files
+  "doctype-first": false,
+  
+  // Tags must be paired
+  "tag-pair": true,
+  
+  // Empty tags must be closed by self
+  "tag-self-close": true 
+  
+  // All special characters must be escaped
+  "spec-char-escape": true,
```

Comment:
Will this cause an issue with say `&` being  on its own. Say `Person & Thing`? If it is going to force escaping that, then this should be false by default. That usage is perfectly valid  and we shouldn't force over-escaping without a good reason.